test/unit/specs/tree.spec.js

```diff
@@ -203,13 +203,12 @@ describe('Tree', () => {
   });
 
   it('current-node-key', done => {
```

Comment:
This is not asynchronous code. `done` is redundant. https://mochajs.org/#asynchronous-code

Reply:
✓